torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -26,6 +26,10 @@ inline Tensor linear(const Tensor& input, const Tensor& weight,
   }
 }
 
+inline Tensor bias(const Tensor& input, const Tensor& bias) {
+  return input + bias;
```

Comment:
In general, it's a good idea to have the functional API call the underlying op (`torch::bias` here) to avoid duplication of the function definition.

Reply:
yes.